eng/ci/build-core-tools-host-artifacts-windows.yml

```diff
@@ -1,89 +1,42 @@
+      ref: refs/tags/release
     - repository: eng
       type: git
       name: engineering
       ref: refs/tags/release
 
-jobs:
-- job: BuildCoreToolsHostWindows
-  displayName: '[Windows] Build CoreToolsHost'
-  pool:
-    vmImage: 'windows-latest'
-
-  variables:
+variables:
   - template: /ci/variables/cfs.yml@eng
 
-  templateContext:
-    outputParentDirectory: $(Build.ArtifactStagingDirectory)
-    outputs:
-    - output: pipelineArtifact
-      displayName: Publish CoreToolsHost packages
-      path: $(Build.ArtifactStagingDirectory)/_coreToolsHostPackagesWindows
-      artifact: _coreToolsHostPackagesWindows
-
-  steps:
-  - pwsh: |
-      Import-Module "./pipelineUtilities.psm1" -Force
-      Install-Dotnet
-    displayName: 'Install .NET 9'
-  - task: DotnetCoreCLI@2
-    displayName: Dotnet Publish (win-x64)
-    inputs:
-      command: publish
-      publishWebProjects: false
-      zipAfterPublish: false
-      arguments: -c Release -r win-x64 -o $(Build.SourcesDirectory)/pkg_output/windows/win-x64
-      workingDirectory: $(Build.SourcesDirectory)/host/src/CoreToolsHost
-
-  - task: DotnetCoreCLI@2
-    displayName: Dotnet Publish (win-arm64)
-    inputs:
-      command: publish
-      publishWebProjects: false
-      zipAfterPublish: false
-      arguments: -c Release -r win-arm64 -o $(Build.SourcesDirectory)/pkg_output/windows/win-arm64
-      workingDirectory: $(Build.SourcesDirectory)/host/src/CoreToolsHost
-
-  - task: CopyFiles@2
-    displayName: Copy files (win-x64)
-    inputs:
-      SourceFolder: $(Build.SourcesDirectory)/pkg_output/windows/win-x64
-      # Publish output will include many other files. We only need func.exe & nethost.dll
-      Contents: |
-        func.exe
-        nethost.dll
-      TargetFolder: $(Build.ArtifactStagingDirectory)/_coreToolsHostPackagesWindows/win-x64
-
-  - task: CopyFiles@2
-    displayName: Copy files (win-arm64)
-    inputs:
-      SourceFolder: $(Build.SourcesDirectory)/pkg_output/windows/win-arm64
-      # Publish output will include many other files. We only need func.exe & nethost.dll
-      Contents: |
-        func.exe
-        nethost.dll
-      TargetFolder: $(Build.ArtifactStagingDirectory)/_coreToolsHostPackagesWindows/win-arm64
-
-  - task: PublishPipelineArtifact@1
-    displayName: 'Publish CoreToolsHost packages artifact'
-    inputs:
-      targetPath: '$(Build.ArtifactStagingDirectory)/_coreToolsHostPackagesWindows'
-      artifact: 'drop-coretools-host-windows'
-      publishLocation: 'pipeline'
\ No newline at end of file
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1es
+  parameters:
+    pool:
+      name: 1es-pool-azfunc
+      image: 1es-windows-2022
+      os: windows
+    sdl:
+      codeql:
+         compiled:
+           enabled: true
+         runSourceLanguagesInSourceAnalysis: true
+    stages:
+      - stage: BuildCoreToolsHost
+        jobs:
+        - template: /eng/ci/templates/official/jobs/build-core-tools-host.yml@self
```

Comment:
You don't _have_ to split this into a separate template. But also fine if you do. It is done that way in other pipelines for organization.